pkg/storage/unified/search/dashboard.go

```diff
@@ -215,3 +335,36 @@ func (s *DashboardDocumentBuilder) BuildDocument(ctx context.Context, key *resou
 
 	return doc, nil
 }
+
+func DashboardFields() []string {
+	baseFields := []string{
+		DASHBOARD_LEGACY_ID,
+		DASHBOARD_SCHEMA_VERSION,
+		DASHBOARD_LINK_COUNT,
+		DASHBOARD_PANEL_TYPES,
+		DASHBOARD_DS_TYPES,
+		DASHBOARD_TRANSFORMATIONS,
+	}
+
+	return append(baseFields, SprinklesFields()...)
+}
+
+func SprinklesFields() []string {
```

Comment:
spinkles is a cute project name, but nobody will know what the heck sprinkles are.  is this more like `userAnalytics`?  A name that describes what this is would be nice for devs.

Reply:
🤷‍♂️ naming is hard. Technically it is Usage Insights. But we also call it Stats in some places

Reply:
I think Stats is reserved for index stats.